trim.production.js

```diff
@@ -31,16 +31,24 @@ function createRecord (title, container, extension, fileData, alternativeContain
     throw new Error('alternativeContainers must be a list of strings: ' + alternativeContainers)
   }
 
+  var body = {
+    Title: title,
+    Container: container,
+    RecordExtension: extension,
+    AlternativeContainers: alternativeContainers,
+    Record: fileData
+  }
+
+  body = R.evolve({
+    Title: R.ifElse(R.isNilOrEmpty, R.always('Untitled Document'), R.replace(/:/g, '-')),
+    RecordExtension: R.replace(/./g, ''),
+    Record: R.pipe(R.split(','), R.last)
```

Comment:
The last string in an array of strings needs to be split if it has a comma in it?

`['a', 'c,b'] -> ['a', ['c', 'b']]` ?